examples/core-hello-world/src/main/scala/com/mesosphere/usi/examples/CoreHelloWorldFramework.scala

```diff
@@ -145,13 +135,11 @@ class CoreHelloWorldFramework(conf: Config) extends StrictLogging {
         assert(failedTasks.isEmpty, s"Found failed tasks: $failedTasks, can't handle them now so will terminate")
 
         logger.info(s"Task status updates for podId: $id: ${taskStatuses}")
-        e
 
       case e =>
-        logger.error(s"Unhandled event: $e") // we ignore everything else for now
-        e
+        logger.warn(s"Unhandled event: $e") // we ignore everything else for now
     }
-    .toMat(Sink.queue())(Keep.right)
+    .toMat(Sink.ignore)(Keep.right)
```

Comment:
since this is a simple example, can we maybe comment on why Sink.ignore is the right think to do here. Because I guess that in every other real framework that probably would not be the case... ?

Reply:
No, it's the other way around - the queue was good for testing (to pull state events out of it) but it also backpressures so not suitable for production.